src/platform/nrfconnect/CHIPPlatformConfig.h

```diff
@@ -40,7 +40,8 @@
 
 #ifdef CONFIG_CHIP_CRYPTO_PSA
 #define CHIP_CONFIG_SHA256_CONTEXT_SIZE sizeof(psa_hash_operation_t)
-#define CHIP_CONFIG_HKDF_KEY_HANDLE_CONTEXT_SIZE sizeof(psa_key_id_t)
+// Alignment to sizeof(PsaHkdfKeyHandle) from crypto/CHIPCryptoPALPSA.h.
+#define CHIP_CONFIG_HKDF_KEY_HANDLE_CONTEXT_SIZE (sizeof(psa_key_id_t) + sizeof(bool))
```

Comment:
Right, but is it a problem? Or you mean updating the comment to be more precise?

Reply:
A bit confusing because `CHIP_CONFIG_HKDF_KEY_HANDLE_CONTEXT_SIZE` is 5 and we cast the handle to a structure that is 8 bytes. But maybe I'm too paranoid so let's leave it as it is :).

Reply:
Ah, now I see what you mean. Yeah, indeed it looks incorrect. The best option would be to use the `sizeof(PsaHkdfKeyHandle)`, but include is not possible. Another option could be to declare the structure of the same content, as `PsaHkdfKeyHandle`, but I'm not sure. Maybe for now it's ok to leave it, as we are going to revert it soon.

Reply:
If it works, then fine! But don't we have any macro or util to align to 8-bytes in Matter?

Reply:
There are cpp macros to generally align the data structure, or return the aligned size of specific type, but to be honest I don't know the macro that would return aligned size from the numerical unaligned size.